samples/WebFormsSample/WebFormsSample.Dynamic.csproj

```diff
@@ -7,6 +7,7 @@
     <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <EnableDefaultContentItems>false</EnableDefaultContentItems>
```

Comment:
complaining about ascx.cs being copied twice, if there is a better way handle let me know. (Don't like above fix)

` Error	NETSDK1022	Duplicate 'Content' items were included. The .NET SDK includes 'Content' items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultContentItems' property to 'false' if you want to explicitly include them in your project file. For more information, see https://aka.ms/sdkimplicititems. The duplicate items were: 'C:\Users\biroj\source\repos\systemweb\systemweb-adapters-ui\samples\WebFormsSample\InsideTestUserControl.ascx.cs'; 'C:\Users\biroj\source\repos\systemweb\systemweb-adapters-ui\samples\WebFormsSample\TestUserControl.ascx.cs'	WebFormsSample.Static	C:\Program Files\dotnet\sdk\8.0.102\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.DefaultItems.Shared.targets	213`

Reply:
not sure - if you can't figure it out, we can follow up in a separate issue after getting the user control compilation working